insights/specs/insights_archive.py

```diff
@@ -294,5 +294,6 @@ class InsightsArchiveSpecs(Specs):
     virt_what = simple_file("insights_commands/virt-what")
     woopsie = simple_file("insights_commands/find_.var.crash_.var.tmp_-path_.reports-_.whoopsie-report")
     yum_list_installed = simple_file("insights_commands/yum_-C_--noplugins_list_installed")
+    yum_list_available = simple_file("insights_commands/yum_--noplugins_list_available")
```

Comment:
@theyoyojo this spec will need to change to be consistent with the spec in `default`.

Reply:
OK, done.